front/app/css/main/components/button.less

```diff
@@ -31,6 +31,86 @@
         padding: 0.2em 0.5em;
     }
 
+    &--clubopening {
```

Comment:
J'ai pris le code css pour ce modèle de bouton sur internet

Reply:
Indique la source alors ?

Reply:
Je l'ai pas mal modifié regarde c'est l'exemple 1 : https://tympanus.net/codrops/2012/09/13/button-switches-with-checkboxes-and-css3-fanciness/